include/boost/geometry/strategies/cartesian/envelope_segment.hpp

```diff
@@ -0,0 +1,64 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2017 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fisikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_CARTESIAN_ENVELOPE_SEGMENT_HPP
+#define BOOST_GEOMETRY_STRATEGIES_CARTESIAN_ENVELOPE_SEGMENT_HPP
+
+#include <boost/geometry/core/tags.hpp>
+#include <boost/geometry/algorithms/detail/envelope/segment.hpp>
+
+namespace boost { namespace geometry
+{
+
+namespace strategy { namespace envelope
+{
+
+template<typename CalculationType>
+class segment_cartesian
```

Comment:
`CalculationType = void`

Regarding the consistency of the names, we'd have `strategy::area::geographic` and `strategy::azimuth::geographic` so my guess as a user would be to expect the name of the CS right after the name of the strategy "category" or algorithm name. So I was thinking about renaming it to `envelope::cartesian_segment` and `envelope::geographic_segment` (we also have intersection::relate_cartesian_segments). And this way it'd also be more pleasant for english speakers. What do you think?